spec/javascripts/services/short_form_application_service_spec.coffee

```diff
@@ -128,7 +128,7 @@ do ->
     describe 'Service setup', ->
       it 'initializes applicant defaults', ->
         expectedDefault = ShortFormApplicationService.applicationDefaults.applicant
-        expect(ShortFormApplicationService.applicant).toEqual expectedDefault
+        expect(ShortFormApplicationService.applicant).toEqual 'blah'
```

Comment:
need to fix this.

Reply:
@christine-sfg this is probably actually a good example, for a common issue that we have which is that jasmine tests (for angular front-end) will often fail silently. If an error is raised in the test or setup and it's not run, the affected tests won't run and the tests will show up as passing.

I found the issue that I attempted to fix here because if you run tests locally, you can see a stack trace error. 

One reason why I recommend always making sure that you can get a test to fail before you get it to pass 😄 

![image](https://user-images.githubusercontent.com/11825994/151272073-ea8afa27-e2cd-4f6e-af0d-be1e38393e7f.png)